Cargo.toml

```diff
@@ -66,10 +66,11 @@ alloy-sol-types = { version = "0.8.12", default-features = false }
 alloy-primitives = { version = "0.8.12", default-features = false }
 
 # Maili
-maili-provider = { version = "0.1.0", default-features = false }
-maili-protocol = { version = "0.1.0", default-features = false }
-maili-registry = { version = "0.1.0", default-features = false }
-maili-rpc-types-engine = { version = "0.1.0", default-features = false }
+maili-provider = { version = "0.1.1", default-features = false }
+maili-protocol = { version = "0.1.1", default-features = false }
+maili-registry = { version = "0.1.1", default-features = false }
+maili-rpc-jsonrpsee = { version = "0.1.1", default-features = false }
+maili-rpc-types-engine = { version = "0.1.1", default-features = false }
```

Comment:
this will be tricky to maintain/release now

what#s the release procedure if both repos depend on each other?

Reply:
it's a step in the migration progress in order to make incremental progress. we will return to it once migration is complete.